src/Stubs/common/DenormalizerInterface.stubphp

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+interface DenormalizerInterface
+{
+    /**
+     * @template     T
+     * @psalm-param  class-string<T> $type
+     * @psalm-return T
+     */
+    public function denormalize($data, string $type, string $format = null, array $context = []);
+
+    /**
+     * @template    T
+     * @psalm-param class-string<T> $type
```

Comment:
template is not needed here, `@psalm-param class-string $type` will work

Reply:
`@param mixed $data` is required for strict mode (before this stub Psalm used the original phpdoc)